storage/database/src/env.rs

```diff
@@ -246,32 +259,22 @@ where
     /// This will open the database [`Table`]
     /// passed as a generic to this function.
     ///
-    #[doc = doc_table_error!()]
-    fn open_db_rw<T: Table>(&self, tx_rw: &Rw) -> Result<impl DatabaseRw<T>, RuntimeError>;
-
-    /// Open all tables in read/iter mode.
-    ///
-    /// This calls [`EnvInner::open_db_ro`] on all database tables
-    /// and returns a structure that allows access to all tables.
+    /// # Errors
+    /// This will only return [`RuntimeError::Io`] on errors.
     ///
-    #[doc = doc_table_error!()]
-    fn open_tables(&self, tx_ro: &Ro) -> Result<impl TablesIter, RuntimeError> {
-        call_fn_on_all_tables_or_early_return! {
-            Self::open_db_ro(self, tx_ro)
-        }
-    }
```

Comment:
## `cuprate_database::Table` 2/4
Since `cuprate_database` has no idea what the user is doing, this function can't exist anymore. What tables should we create? That is up to the user now.